solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -255,15 +247,12 @@ private void dispatch(
       Action result = call.call();
       switch (result) {
         case PASSTHROUGH:
-          getSpan(request).log("SolrDispatchFilter PASSTHROUGH");
```

Comment:
Not sure what this log call did, are we removing something from the trace here that we had before?

Reply:
replaced with 'addEvent', as discussed above